### PR TITLE
Added error handling.

### PR DESCRIPTION
**Pull request by MGGSK**
When an unhandled exception gets thrown the app no longer crashes. Now the user gets a notification with the exception.

## Description
1. Added a event handler to App.UnhandledException.
2. Written code to create and show the notification. 

## Motivation and Context
The app crashes a lot.

## How Has This Been Tested?
While testing I added a button that throws an exception to test my code.

## Screenshots:
image

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/App.xaml.cs [hunk 1]
```diff
@@ -24,6 +24,7 @@
 using System.Text;
 using Windows.System;
 using System.Runtime.InteropServices;
+using Microsoft.Toolkit.Uwp.Notifications;
```
Comment (karkarl) [inline WinUIGallery/App.xaml.cs:27]: Would you be able to use the WinAppSDK [Notifications](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/app-notifications/app-notifications-quickstart?tabs=cs) instead from the Community Toolkit?

Comment (MGGSK) [reply WinUIGallery/App.xaml.cs:27]: I like using the community toolkit more so I used it.

Comment (MGGSK) [reply WinUIGallery/App.xaml.cs:27]: Can it be like this?    
using Microsoft.Windows.AppNotifications;
using Microsoft.Windows.AppNotifications.Builder;
       //Create the notification.
            var notification = new AppNotificationBuilder()
                .AddText("An exception was thrown.")
                .AddText($"Type: {e.Exception.GetType()}")
                .AddText($"Message: {e.Message}")
                .AddText($"HResult: {e.Exception.HResult}")
                .BuildNotification();

            //Show the notification
            AppNotificationManager.Default.Show(notification);

Comment (MGGSK) [reply WinUIGallery/App.xaml.cs]: Changed it.

File: WinUIGallery/WinUIGallery.csproj [hunk 2]
```diff
@@ -83,6 +83,30 @@
         <AppxBundle>Never</AppxBundle>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug-Unpackaged|x86'" />
```
Comment (karkarl) [inline WinUIGallery/WinUIGallery.csproj:86]: Are changes here intentional? If so, what is it for?

Comment (MGGSK) [reply WinUIGallery/WinUIGallery.csproj]: > Are changes here intentional? If so, what is it for?

Fixed it.

**Review by karkarl — APPROVED** [review]

**Pull request merged**